app/src/main/resources/db/migration/V1__init.sql

```diff
@@ -0,0 +1,78 @@
+create table if not exists users
+(
+    user_id    bigint       not null primary key,
+    username   varchar(25)  not null,
+    password   varchar(255) not null,
+    user_type  varchar(16)  not null check (user_type in ('ROLE_USER', 'ROLE_ADMIN')),
+    created_at TIMESTAMP(6) not null,
+    updated_at TIMESTAMP(6) not null
+) ENGINE = InnoDB
+  DEFAULT CHARSET = utf8mb3;
```

Comment:
utf8mb3는 곧 사라진다고 해서 urf8mb4로 바꾸는게 좋을거 같습니다!

```suggestion
  DEFAULT CHARSET = utf8mb4;
```